reana_server/reana_admin/check_workflows.py

```diff
@@ -146,73 +164,144 @@ def check_workflows(
 
     time_range_message = f"{date_start} - {date_end if date_end else 'now'}"
 
-    if filtered_workflows.count() == 0:
+    total_workflows = filtered_workflows.count()
+
+    if total_workflows == 0:
         click.secho(
-            f"No workflows found matching time range {time_range_message}.\nPlease, adjust your filter conditions.",
+            f"No workflows found matching time range {time_range_message}.",
             fg="red",
         )
-        sys.exit(1)
+        return [], [], total_workflows
```

Comment:
Before we were exiting with status code 1, while now we don't. Is this supposed to be treated as an error or not? What do you think?

Reply:
It should be an error. If no workflows are found, it can mean that no workflows were run during that time. If we use this tool regularly, it might return "FAILED" when there were just zero workflows run.